src/JobItem.js

```diff
@@ -0,0 +1,21 @@
+import  React, { Component } from 'react';
+import jobUtils from './jobUtils';
+import './JobItem.css';
+
+class JobItem extends Component {
```

Comment:
If this won't have any local state, could we just make this a functional component?